library/src/main/java/com/getstream/sdk/chat/rest/core/Client.java

```diff
@@ -130,12 +131,15 @@ public void setState(ClientState state) {
     // Main Params
     private String apiKey;
 
+    private ApiClientOptions apiClientOptions;
```

Comment:
what about we this: 

remove from `getWebSocketDisconnectDelay` ApiClientOptions
add a private field to Client to store the delay internal
expose delay field with a setter

With this approach we remove an API breaking change and still have that configurable (most people won't do that)

Reply:
- Removed getWebSocketDisconnectDelay
- We already have private field with default delay which set in constructor
- Added setter